src/java.base/share/classes/java/lang/Module.java

```diff
@@ -229,7 +230,7 @@ public ModuleDescriptor getDescriptor() {
     /**
      * Returns the module layer that contains this module or {@code null} if
      * this module is not in a module layer.
-     *
+     * <p>
```

Comment:
This will break up the first paragraph, I don't think this PR should be changing that.